CODE_OF_CONDUCT.md

```diff
@@ -31,6 +31,50 @@ Examples of unacceptable behavior by participants include:
 - Other conduct which could reasonably be considered inappropriate in a
   professional setting
 
+### Guide to Maintaining These Standards
+
+When interacting in our community, strive to maintain a friendly, empathetic tone. This includes, but is not limited to, interactions on Discord, GitHub, and Twitter.
+
+"Tone" refers to word, punctuation, and conversational choices that determine how your message _feels_ to others.
```

Comment:
```suggestion
"Tone" refers to word, punctuation, emojis, gifs, and conversational choices that determine how your message _feels_ to others.
```

There were many times when I was trolled by someone's use of a reaction emoji or witnessed people offended by a gif that was added to a message. Think about, for instance, any emoji with innuendos or gifs known to be sexist/racist.